config/tirpc.m4

```diff
@@ -0,0 +1,17 @@
+dnl
+dnl musl support
+dnl
+
+AC_DEFUN([ZFS_AC_CONFIG_USER_TIRPC], [
+AH_TEMPLATE([WITH_TIRPC],
```

Comment:
`AH_TEMPLATE` does not appear to be used anywhere in the existing ZoL autotools code. This autotools code is failing the buildbot, so this probably should be made consistent with the rest of it to try to fix that.